src/main/java/nextstep/lms/domain/LmsUser.java

```diff
@@ -0,0 +1,66 @@
+package nextstep.lms.domain;
+
+import java.time.LocalDateTime;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class LmsUser {
+    private static final AtomicLong idGenerator = new AtomicLong(1);
+
+    private Long id;
+
+    private String userId;
+
+    private String password;
+
+    private String name;
+    private LmsUserRole role;
+
+    private LocalDateTime createdAt;
+
+    private LocalDateTime updatedAt;
+
+    private LmsUser() {
+    }
+
+    private LmsUser(String userId, String password, String name, LmsUserRole role) {
+        this(idGenerator.getAndIncrement(), userId, password, name, role, LocalDateTime.now(), null);
+    }
+
+    private LmsUser(Long id, String userId, String password, String name, LmsUserRole role, LocalDateTime createdAt, LocalDateTime updatedAt) {
+        this.id = id;
+        this.userId = userId;
+        this.password = password;
+        this.name = name;
+        this.role = role;
+        this.createdAt = createdAt;
+        this.updatedAt = updatedAt;
+    }
+
+    public static LmsUser createAdmin(String userId, String password, String name) {
+        return new LmsUser(userId, password, name, LmsUserRole.ADMIN);
+    }
+
+    public static LmsUser createUser(String userId, String password, String name) {
+        return new LmsUser(userId, password, name, LmsUserRole.NORMAL);
+    }
+
+    public boolean isAdmin() {
+        return role == LmsUserRole.ADMIN;
```

Comment:
```suggestion
        return role.isAdmin();
```

서비스를 개발하다보면 `role == LmsUserRole.ADMIN;` 이러한 로직이 여러군데 나온다고 가정해볼께요!
그렇다면, 중복코드가 발생하고 LmsUserRole 를 수정하기 힘들어지지 않을까요? 
외부에서 값을 가져다가 사용하고 있으니까요!

객체를 만든 만큼 상태 직접 비교하도록 하면 좋을것 같아요!